src/v0/sources/shopify/transform.js

```diff
@@ -150,6 +163,15 @@ const processEvent = async (inputEvent, metricMetadata) => {
   if (message.type !== EventType.IDENTIFY) {
     if (useRedisDatabase) {
       await setAnonymousIdorUserIdFromDb(message, metricMetadata);
+      if (message.anonymousId === "discard") { // cart create event generated by us. Hence discarding
+        return {
+          outputToSource: {
+            body: Buffer.from('OK').toString('base64'),
+            contentType: 'text/plain',
+          },
+          statusCode: 200,
+        }
+      }
```

Comment:
What is this scenario?

Reply:
Why can't we Directly stop allowing `carts_create` events?

Reply:
How can we directly stop allowing carts_create event? Like we need some information to know whether it is created by user or not

Reply:
I suppose we do not support `carts_create` event in Shopify client side but rather all `carts_create` events are generated by us only and internally used. Isn't that the case?

Reply:
okay . Removed